src/routes/v1/friendships/mod.rs

```diff
@@ -1,4 +1,5 @@
 pub mod errors;
 pub mod get;
+pub mod history;
```

Comment:
is it ok to add this new file? or should I move everything to `friendships/get`?